### ISSUE TITLE
EDIT ME Include the app name

### ISSUE THREAD

**Issue by PGSCOM** [Botspot/pi-apps#1976]
### What is the name of the app?

Netflix client

### About the app

The Netflix app or something similar for download movies and view it in the rpi4 offline. An idea it´s the app emulated with wine.

### Confirmations

- [X] I have confirmed that this app is legal and not piracy.
- [X] I have confirmed that this app has never been discussed in https://github.com/Botspot/pi-apps/issues and https://github.com/Botspot/pi-apps/pulls, and it is not in the Pi Apps app list.
- [X] I have confirmed that this app can run on Raspberry Pi.

**Comment by Crilum** [comment]
This is probably not realistically possible..

**Comment by PGSCOM** [comment]
> This is probably not realistically possible..
> 
> Please change the issue title to match your request.

I know one way which is with a [Kodi plugin](https://github.com/asciidisco/plugin.video.netflix) but this need internet.

**Comment by Crilum** [comment]
I don't think you can download Netflix content (without an official Netflix app)..

**Comment by Crilum** [comment]
That Kodi plugin is depreciated, this is the actively developed: https://github.com/CastagnaIT/plugin.video.netflix/
Here are install instructions: https://github.com/CastagnaIT/plugin.video.netflix/wiki/How-install-the-addon
I'm not sure how this could be made into a script though, because you have to open Kodi to install the repo..

**Comment by theofficialgman** [comment]
I don't think that plugin will work on arm64 (or even armhf). it almost certainly ships only x86_64 widefine as part of the plugin. without it, nothing will work

**Comment by Crilum** [comment]
> I don't think that plugin will work on arm64 (or even armhf). it almost certainly ships only x86_64 widevine as part of the plugin. without it, nothing will work
> 
> better to wait till ARM64 userspace chromebooks are released so people can patch the arm64 winevine from chromeos to work on our linux browsers

I don't have Netflix, so I can't test it, but I got to the login part just fine.. Obviously that's not the part that uses Widevine, but who knows?
